### PR TITLE
rg-151: User avatar upload

### PR DESCRIPTION
**Pull request by vlad-blip**
[Take a look at Demo](https://www.loom.com/share/fc209e8306bd48be83baf507a5a8cbce?sid=65bbffc8-e04d-4d78-9c33-9ca5356ae6ea)

### CODE REVIEW

**Review by MaxPoplavskiy — APPROVED** [review]

File: backend/src/bundles/profile/profile.repository.ts [hunk 1]
```diff
@@ -12,6 +12,20 @@ class ProfileRepository extends AbstractRepository<
     }: Record<'profileModel', typeof ProfileModel>) {
         super(profileModel);
     }
+
+    public async uploadAvatar({
```
Comment (honkster-nazarii) [inline backend/src/bundles/profile/profile.repository.ts:16]: confusing name for repository method, consider changing it to more fitting name

**Review by honkster-nazarii — CHANGES_REQUESTED** [review]
Would be cool to have a short demo

**Comment by vlad-blip** [comment]
> Would be cool to have a short demo

Do you mean a video demo? Gonna add it tomorrow.

**Review by Icerock10 — APPROVED** [review]

**Review by honkster-nazarii — APPROVED** [review]

**Comment by Icerock10** [comment]
conflicts

**Review by MaxPoplavskiy — APPROVED** [review]

**Review by honkster-nazarii — APPROVED** [review]

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

**Pull request merged**